src/TSTransformer/classes/TransformState.ts

```diff
@@ -213,7 +213,7 @@ export class TransformState {
 				for (let i = 1; i < this.runtimeLibRbxPath.length; i++) {
 					expression = luau.create(luau.SyntaxKind.MethodCallExpression, {
 						expression,
-						name: "WaitForChild",
+						name: "FindFirstChild",
```

Comment:
This would actively break usage of the RuntimeLib from ReplicatedFirst. While currently a warning, it's not invalid and there are some genuine use-cases.
In general, I think it'd make more sense to look into shimming `WaitForChild` in Lune, rather than changing compiler behaviour.

Reply:
I think these are hacks for this branch that will be swapped back over to current behavior once Lune supports these features (like `WaitForChild`)